### ISSUE TITLE
MSI for Microsoft.Azure.ServiceBus client is missing

### ISSUE THREAD

**Issue by SeanFeldman** [Azure/azure-service-bus#135]
[The sample](https://github.com/Azure/azure-service-bus/tree/master/samples/DotNet/Microsoft.ServiceBus.Messaging/ManagedServiceIdentity) is available for the old client, but not the new one.

**Comment by SeanFeldman** [comment]
cc @ChristianWolf42

**Comment by ChristianWolf42** [comment]
@SeanFeldman Would you mind adding one to https://github.com/Azure/azure-service-bus/tree/master/samples/DotNet/Microsoft.Azure.ServiceBus?

**Comment by SeanFeldman** [comment]
Assign to me @ChristianWolf42

**Comment by ChristianWolf42** [comment]
@SeanFeldman thanks a lot

**Comment by SeanFeldman** [comment]
This will require Microsoft.Azure.ServiceBus version 3.0.0-preview-01 and later.
Is that ok @ChristianWolf42 ?

**Comment by ChristianWolf42** [comment]
@SeanFeldman yes, I think that is where we released the msi support in if I remember right.

**Comment by ChristianWolf42** [comment]
@SeanFeldman, I don't want to get greedy but could you add one for RBAC too? :-)

**Comment by leftside** [comment]
@ChristianWolf42 which article are you referring to? I'm following along as I want to do PoC for my company based upon the article that @SeanFeldman wrote a couple of months ago:
https://weblogs.asp.net/sfeldman/azure-service-bus-with-managed-service-identity

**Comment by ChristianWolf42** [comment]
@leftside this blog will lead you to documentation and Github repo with regarding samples:
https://blogs.msdn.microsoft.com/servicebus/2017/12/20/azure-service-bus-managed-service-identity-msi-and-role-based-access-control-rbac-preview-released/
